scripts/git/post-merge.git-hook

```diff
@@ -39,6 +39,7 @@ check_for_diffs "pre-push"
 check_for_diffs "pre-commit"
 check_for_diffs "post-merge"
 
-check_for_script_update "git-merge-forward.sh"
-check_for_script_update "git-pull-all.sh"
-check_for_script_update "git-push-all.sh"
+for file in ls $git_toplevel/scripts/ ; do
```

Comment:
This won't work -- it will first set file to "ls" and then to "$ggit_toplevel/scripts/".

Please make sure you test your code; it will save time in the long run.